source/initHQ/fortifyFOB2.sqf

```diff
@@ -0,0 +1,90 @@
+_centerPos = _this  select 0;
+_fob = _this select 1;
+
+_groupGuard = createGroup WEST;
+
+// AA on the roof
+_aapod = createVehicle ["B_static_AA_F", [0,0,0], [], 0, "NONE"];
+_aapod allowdamage false;
+_aapod setdamage 0;
+_aapod setpos [_centerpos select 0, _centerpos select 1,(_centerpos select 2)+3.109];
+
+_fobguard1 = _groupGuard createUnit ["B_Soldier_F", _centerPos, [], 0, "FORM"]; 
+_fobguard1 moveinGunner _aapod;
+
+// GMG
+_gl1pod = createVehicle ["B_GMG_01_high_F", [0,0,0], [], 0, "NONE"];
+_gl1pod allowdamage false;
+_gl1pod setdamage 0;
+_gl1pod setpos [(_centerpos select 0)-7, (_centerpos select 1)+5.5];
```

Comment:
Through the rest of this file we used names like `atpod1` but here we went `gl1pod`. Can this get renamed to `glPod1` for consistency please?